monero-outreach-docs/translations/nl/PSA/files/exchange.md

```diff
@@ -0,0 +1,30 @@
+# Ken de veiligheid van beurzen en wisselkantoren
```

Comment:
# Ken de veiligheid van beurzen en wisselkantoren

Beurzen kunnen handig zijn om Monero te (ver)kopen, maar er zijn onderlinge verschillen.

## Onthou het volgende om je XMR veilig te houden bij het gebruik van een beurs:

1. Doe je eigen research. Probeer de volgende vragen te beantwoorden:

- a. Zijn er positieve reviews van onafhankelijke bronnen? Zijn ze betrouwbaar?

- b. Is de beurs ooit aangevallen? Was de reactie daarop gepast?

- c. Hoe lang is hij al operationeel?

- d. Zijn de oprichters/managers bekend? Hebben deze individuen of entiteiten een goede reputatie?

- e. Kan je je privacy behouden? Hoef je geen persoonlijke informatie te delen?

- f. Hanteert de website beveiligingsmaatregelen zoals "captcha" en "2FA" (Twee-Factor-Authenticatie)? Zijn de beveilingscerificaten legitiem en start het adres met "https://"?

- g. Indien je belastingaangifte wil doen: kan je bewijsstukken van je transacties zonder moeite downloaden? 

2. Beurzen creëren een (eigen) wallet in jouw plaats. Waarschijnlijk heb je geen toegang tot de hersteltekst. Als de beurs wordt aangevallen of bedrog blijkt te zijn, verlies je alles in deze wallet. Een goede vuistregel is er nooit langdurig XMR te bewaren: indien je het niet op zeer korte termijn spendeert, breng het dan over naar een wallet die je zelf beheert.

3. Vooraleer je een significant bedrag overbrengt naar een beurs dien je best een testtransactie te plaatsen. Gebruik een testbedrag dat je kan veroorloven te verliezen.

4. Kijk altijd het zend- en ontvangstadres na bij elke XMR-transactie. Doe dit zelfs als je kopieert en plakt, want er bestaan aanvallen waarbij je crypto-adres wordt veranderd. Moest zulke malware je apparaat hebben besmet, voldoet simpel nakijken wat je plakt om je te beschermen.

5. Op termijn zullen gedecentraliseerde beurzen gemeengoed worden. Deze liggen qua ethos in the verlengde van de gedecentraliseerde cryptovaluta Monero. Overweeg ze te gebruiken, maar doe je research!